posts/inside-rust/2024-09-04-electing-new-project-directors.md

```diff
@@ -0,0 +1,86 @@
+Nominees can be anyone in the project and do not have to be a member of the team who nominates them.
+
+The candidate gathering process will be open until October 9, at which point each team's Council Representative will share their team's nominations and reasoning with the whole Leadership Council.
+At this point, the Council will confirm with each of the nominees that they are willing to accept the nomination and fill the role of Project Director.
+Then the Council will publish the set of candidates.
+
+This then starts a ten day period where members of the Rust Project are invited to share feedback on the nominees with the Council.
+This feedback can include reasons why a nominee would make a good project director, or concerns the Council should be aware of.
+
+The Council will announce the set of nominees by October 11 and the ten day feedback period will last until October 21.
+Once this time has passed, we will move on to the election phase.
+
+## Election
+
+The Council will meet during the week of October 21 to complete the election process.
+In this meeting we will discuss each candidate and once we have done this the facilitator will propose a set of two of them to be the new Project Directors.
+The facilitator puts this to a vote, and if the Council unanimously agrees with the proposed pair of candidates then the process is completed.
+Otherwise, we will give another opportunity for council members to express their objections and we will continue with another proposal.
+This process repeats until we find two nominees who the Council can unanimously consent to.
+The Council will then confirm these nominees through an official vote.
+
+Once this is done, we will announce the new Project Directors.
+In addition, we will contact each of the nominees, including those who were not elected, to tell them a little bit more about what we saw as their strengths and opportunities for growth to help them serve better in similar roles in the future.
+
+## Timeline
+
+This process will continue through all of September and into October.
+Below are the key dates:
+
+* Candidate nominations due: October 9
+* Candidates published: October 11
+* Feedback period: October 11 - October 21
+* Election meeting: Week of October 21
+* First board meeting starting the new term: November 12
+
+After the election meeting happens, the Rust Leadership Council will announce the results and the new Project Directors will assume their responsibilities.
+
+## Acknowledgements
+
+A number of people have been involved in designing and launching this election process and we wish to extend a heartfelt thanks to all of them!
+We'd especially like to thank the members of the Project Director Election Proposal Committee: Jane Losare-Lusby, Eric Holk, and Ryan Levick.
+Additionally, many members of the Rust Community have provided feedback and thoughtful discussions that led to significant improvements to the process.
+We are grateful for all of your contributions.
```

Comment:
Should we rephrase acknowledgements now that we're mostly repeating the process? Maybe we can point to recent updates to the process and in-progress work on resolving/clarifying affiliation concerns?

Reply:
I thought about changing this, but wasn't sure what to change. I pushed an update here to make it a little more current.